tutorials/cp-cf-processvisibility-config-samplescenario/cp-cf-processvisibility-config-samplescenario.md

```diff
@@ -139,19 +139,19 @@ In this step, you can process the data of the activated business scenario using
 
 In this step, you will be able to track the progress of the sample design-to-operate business scenario using the visual representation of the Process Workspace application.
 
-1. Log on to the **Process Workspace** tile on SAP Fiori launchpad.
+1. Click on the **Process Workspace** tile by navigating to the home screen.
```

Comment:
We don't write "click on the [thing to be clicked]". Just write "click the [thing to be clicked]".